app/models/sign.rb

```diff
@@ -7,6 +7,10 @@ class Sign < ApplicationRecord
   PERMITTED_IMAGE_CONTENT_TYPE_REGEXP = %r{\Aimage/.+\Z}
   REFERRED_TOPIC = %r{\A/topics/\d{1,3}\Z}
   MAXIMUM_FILE_SIZE = 250.megabytes
+  SORT_OPTIONS = {
+    alphabetical: "word ASC",
```

Comment:
i suggest keeping these values as the hash arguments, this means if it ends up that any eager loading gets joined to the request this sort is involved in, then it will sort on the created_at column from the correct table, rather than getting confused.
`alphabetical: { word: :asc},`